ansible/requirements.txt

```diff
@@ -1,4 +1,4 @@
-ansible==2.2.1
+ansible==2.4
```

Comment:
Anyone see an issue with this? I'm going to run a deploy to the couch machine with this to confirm it doesn't cause any problems.

Reply:
perhaps some deprecated warnings will become errors now, but i think it's a good to go for it even if it requires extra work to update some playbooks

Reply:
Hmm, so looks like we can't do this because there's an active bug in the monit module in 2.4 that prevents monit 5.18 and below from working; we're on monit 5.6 and that's the latest available in apt.